examples/example_basicusage.ipynb

```diff
@@ -0,0 +1,424 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a3681317-4fd7-434a-9ac5-de994c216826",
+   "metadata": {},
+   "source": [
+    "# Installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8dad5bd8-d71d-4b37-a513-88689c9bb504",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install Opti-HPLC-Handler # Inital installation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "97587f1d-4223-48ab-853e-793e93c5bdff",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install Opti-HPLC-Handler --upgrade # Update"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef4c3950-5578-4d87-8c11-184092f12e9f",
+   "metadata": {},
+   "source": [
+    "# Importing and Login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0990b0e4-2fcd-4e60-ae65-c76a978fd74a",
+   "metadata": {},
+   "source": [
+    "Choose project and log in"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "c9673853-ff9b-464e-a498-ba74d0649fac",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Please enter the password for user slcb:  ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "from OptiHPLCHandler import EmpowerHandler\n",
+    "handler=EmpowerHandler(project=\"SLCB_2023_DEV\",address=\"http://epdkhqr01018.corp.novocorp.net:3076/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b90cac75-a886-4584-9eb9-6ca474b0845f",
+   "metadata": {},
+   "source": [
+    "# Connection Token "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 36,
+   "id": "35723a65-31f4-4981-8238-cc53c1961293",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1laWQiOiI5Yzg0OGFlNC01ZWU3LTQwMjYtYTRhOS1hMzkyYTFlY2JjOTQiLCJpc3MiOiJXYXRlcnMuRW1wb3dlci5XZWIuQVBJIiwidW5pcXVlX25hbWUiOiJFM0ZSNU5PTkdYUFxcU0xDQl8yMDIzX0RFVlxcc2xjYiIsImlhdCI6MTY5MjA5Nzc1OSwiZXhwIjoxNjkyMzk3NzU5LCJhdWQiOiJXYXRlcnMifQ.Jd6jkxhoK0gmZvUjhZkKqea2owV5Xc4T-2RgaceivQQ'"
+      ]
+     },
+     "execution_count": 36,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "handler.connection.token"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5539be2f-42bd-4644-b00c-5378c19840f5",
+   "metadata": {},
+   "source": [
+    "# Get"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "02caa267-45a0-4e1e-a891-ce23bc83a18a",
+   "metadata": {},
+   "source": [
+    "Get list of methods associated with chosen project. Here I have only pulled the first 5 results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "ba651983-5b47-421a-9e97-138fe41d6004",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['Fus_20230504_GSNO_EDTA',\n",
```

Comment:
Is any of these results confidential, or "for internal use only"? e.g. method names, HPLC names, node names?